scripts/PBAB+Collabs/zksync-testnet/hardhat.config.ts

```diff
@@ -0,0 +1,30 @@
+require("@matterlabs/hardhat-zksync-deploy");
+require("@matterlabs/hardhat-zksync-solc");
+
```

Comment:
general question here - how do you define your zksync wallet private keys (for running deployments, etc.)?